backend/LexBoxApi/GraphQL/RefreshProjectMembershipInterceptor.cs

```diff
@@ -8,17 +8,21 @@
 
 namespace LexBoxApi.GraphQL;
 
-public class RefreshJwtProjectMembershipMiddleware(FieldDelegate next)
+public class RefreshJwtProjectMembershipMiddleware(FieldDelegate next, ILogger<RefreshJwtProjectMembershipMiddleware> logger)
```

Comment:
@hahn-kev You didn't end up using the logger. Did you intend to?

Reply:
I used it for checking out the duplication issue. We don't need to keep it around